main/nfs-utils/APKBUILD

```diff
@@ -1,11 +1,14 @@
 # Contributor: Carlo Landmeter <clandmeter@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=nfs-utils
-pkgver=2.1.1
-_basever=$pkgver
-pkgrel=4
+pkgver=2.2.1
+# Get the release commit hash from
+# http://git.linux-nfs.org/?p=steved/nfs-utils.git;a=summary
+_commit=a867b45e2cd82c672c878bb6cc2cfd655aa7c6b4
```

Comment:
Why? There are tags in the upstream repository…

Reply:
I tried to use the release tag but could not figure out how to get the archive... :/

Reply:
I figured out a way to use the tag:
http://git.linux-nfs.org/?p=steved/nfs-utils.git;h=refs/tags/nfs-utils-2-3-1;a=snapshot;sf=tgz

I will make the changes later and bump the version.

gitweb is annoying...

Reply:
Yeah, gitweb is quite bad.